main.go

```diff
@@ -42,7 +47,7 @@ type ghClient struct {
 }
 
 func main() {
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```

Comment:
We have more repositories to process now, so we're increasing the timeout value to 5 minutes.